src/ocrd_utils/config.py

```diff
@@ -145,6 +145,16 @@ def _ocrd_download_timeout_parser(val):
         description="Default address of Processing Server to connect to (for `ocrd network client processing`).",
         default=(True, ''))
 
+config.add("OCRD_NETWORK_CLIENT_POLLING_TIMEOUT",
```

Comment:
Don't forget to add these to `src/ocrd/cli/__init__.py` for `--help` documentation.

Reply:
Fixed with [0136db0](https://github.com/OCR-D/core/pull/1269/commits/0136db04e39269f77dfab3b71e46e43fa189e839).